### PR TITLE
console: add help button and move about page to settings

### PR DESCRIPTION
**Pull request by rikinsk**
### Description

- moved About page to Settings section
- standardized headings in Setting section tabs
- Replaced header `?` button with a `HELP` button that links to https://hasura.io/help

### Changelog

- [x] `CHANGELOG.md` is updated with user-facing content relevant to this PR.

### Affected components

- [x] Console

### Steps to test and verify

https://hge-ci-pull-4848.herokuapp.com/console

### CODE REVIEW

File: console/src/components/Main/Main.js [hunk 1]
```diff
@@ -776,97 +765,23 @@ class Main extends React.Component {
                 {renderProPopup()}
               </div>
               <Link to="/settings">
-                <div className={styles.helpSection + ' ' + styles.settingsIcon}>
-                  {getMetadataStatusIcon()}
-                  {getSettingsSelectedMarker()}
+                <div className={styles.supportSection}>
+                  <div className={styles.helpSection}>
```
Comment (beerose) [inline console/src/components/Main/Main.js:769]: What about _merging_ it into one div:
```
<div className={`${styles.supportSection} ${styles.helpSection}`}>
```
?

File: console/src/components/Main/Main.js [hunk 2]
```diff
@@ -776,97 +765,23 @@ class Main extends React.Component {
...
-                >
-                  <i className={styles.question + ' fa fa-question'} />
+              <a
+                id="help"
+                href={'https://hasura.io/help'}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <div className={styles.supportSection}>
+                  <div className={styles.helpSection}>HELP</div>
```
Comment (beerose) [inline console/src/components/Main/Main.js:782]: Same here:
```
<div className={`${styles.supportSection} ${styles.helpSection}`}>
```

File: console/src/components/Services/Settings/About/About.js [hunk 3]
```diff
@@ -134,11 +134,13 @@ class About extends Component {
...
     return (
-      <div className={`container-fluid ${styles.full_container}`}>
+      <div
+        className={`${styles.clear_fix} ${styles.padd_left} ${styles.padd_top} ${styles.metadata_wrapper} container-fluid`}
+      >
         <div className={styles.subHeader}>
           <Helmet title={'About | Hasura'} />
           <h2 className={styles.headerText}>About</h2>
-          <div className={styles.wd60}>
+          <div>
```
[the remark this answers is not in the document]
Comment (rikinsk) [reply console/src/components/Services/Settings/About/About.js]: Done

File: console/src/components/Main/Main.js [hunk 4]
```diff
@@ -208,11 +208,10 @@ class Main extends React.Component {
...
     const logo = require('./images/white-logo.svg');
-    const github = require('./images/Github.svg');
-    const discord = require('./images/Discord.svg');
-    const mail = require('./images/mail.svg');
-    const docs = require('./images/docs-logo.svg');
-    const about = require('./images/console-logo.svg');
+    // const github = require('./images/Github.svg');
+    // const discord = require('./images/Discord.svg');
+    // const mail = require('./images/mail.svg');
+    // const docs = require('./images/docs-logo.svg');
```
[the remark this answers is not in the document]
Comment (rikinsk) [reply console/src/components/Main/Main.js]: Done

File: console/src/components/Main/Main.js (hunk 1 above)
Comment (rikinsk) [reply console/src/components/Main/Main.js]: This was due to the way the current css was structured and I didnt want to go down that hole. Updated it though.

File: console/src/components/Main/Main.js (hunk 2 above)
Comment (rikinsk) [reply console/src/components/Main/Main.js]: Done

**Pull request merged**